### ISSUE TITLE
Bug on LITD instances // unable to use genseed API

### ISSUE THREAD

**Issue by weltitob** [lightninglabs/lightning-terminal#805]
JSON Response: {'code': 2, 'message': 'lnd is not ready for: /lnrpc.WalletUnlocker/GenSeed', 'details': []} // The genSeed API service is uncallable for LITD instances after startup eventough it should be the way hwo to start them up - my code:

```
import base64
import json
import requests

REST_HOST = '54.198.183.180:8443'
#TLS_PATH = 'LND_DIR/tls.cert'

url = f'https://{REST_HOST}/v1/genseed?"aezeed_passphrase"=Test'

data = {
    #'aezeed_passphrase': base64.b64encode(mnemonic_seed.encode()).decode('utf-8'),
    #'seed_entropy': base64.b64encode(b'').decode('utf-8'),
}

headers = {
    'Content-Type': 'application/json'
}

r = requests.get(url, headers=headers, data=json.dumps(data), verify=False)


# Try to print JSON response if available
try:
    json_response = r.json()
    print("JSON Response:", json_response)

except requests.exceptions.JSONDecodeError:
    print("Response is not a valid JSON: ", r)
```

endpoint url = f'https://{REST_HOST}/v1/state' retruns {'state': 'NON_EXISTING'}

However: Using lnclli create works completly fine on the same instance, this issue was also reported after I mentioned it in the forum by another user
![image](https://github.com/user-attachments/assets/58362c6a-77af-4101-8aef-0c5219ca9fe7)

Please can someone help me with this as it is crucial for my application development to continue

**Comment by weltitob** [comment]
I need help on this issue please. Might also be a failure in me calling the api but I coudn't find anything so far

**Comment by guggero** [comment]
What are you trying to achieve? Generate a completely new seed? Because that's what the `genseed` call is for. It's completely stateless and doesn't affect the wallet state at all.
But from your example code it looks to me like you're trying to create a wallet with an existing seed? That would then be `InitWallet` (`/v1/initwallet`) call.

**Comment by weltitob** [comment]
> What are you trying to achieve? Generate a completely new seed? Because that's what the `genseed` call is for. It's completely stateless and doesn't affect the wallet state at all. But from your example code it looks to me like you're trying to create a wallet with an existing seed? That would then be `InitWallet` (`/v1/initwallet`) call.

Well yes I'm basically just trying to intialize a user wallet after creating a completly new litd cloud instance >> {'code': 2, 'message': 'lnd is not ready for: /lnrpc.WalletUnlocker/InitWallet', 'details': []} I get the same error for init_wallet unfortunatly. How do I get my lnd "ready" to even call these api endpoints without using the cli? I need to spin up cloud containers and then want to be able to call them initalize wallets instantly without having a manual lncli script running or smth. It should be possible from API too, to create and unlock a wallet, right? It's just that these endpoints all give me the response that they're not ready. 

`
import base64
import json
import requests

REST_HOST = '54.144.88.180:8443'
TLS_PATH = 'LND_DIR/tls.cert'

url = f'https://{REST_HOST}/v1/initwallet'

# Example 24-word mnemonic seed

data = {
  'wallet_password': base64.b64encode(b'your_wallet_password_here').decode('utf-8'),
  'cipher_seed_mnemonic': *****************,
  'aezeed_passphrase': base64.b64encode(b'your_aezeed_passphrase_here').decode('utf-8'),
  'recovery_window': 0,  # Example value, adjust as needed
  'channel_backups': None,  # Example value, provide actual ChanBackupSnapshot if available
  'stateless_init': False,  # Example value, set to True or False as needed
  'extended_master_key': '',  # Example value, provide actual extended master key if available
  'extended_master_key_birthday_timestamp': 0,  # Example value, provide actual timestamp if available
  'watch_only': None,  # Example value, provide actual WatchOnly details if available
  'macaroon_root_key': base64.b64encode(b'********************').decode('utf-8'),
}

r = requests.post(url, data=json.dumps(data), verify=False)
print(r.json())

`

**Comment by guggero** [comment]
Thank you for your patience. I finally found time to reproduce this, and you're right, this is a bug in `litd`.
Fix is incoming here: #806.

Btw, to format a multi-line piece of code, you need to use three back ticks (a single one is just for inline code).

**Comment by weltitob** [comment]
@guggero Wow, thanks for acting and no worries at all! :)
